core/main.py

```diff
@@ -100,17 +101,44 @@ def parse_args(args):
         parents=[base_subparser],
         help="Show upstream dependencies for a model",
     )
-    upstream_depencies_sub.set_defaults(cls=show_dependencies_task.ShowDependenciesTask, which="show_upstream")
+    upstream_depencies_sub.set_defaults(
+        cls=show_dependencies_task.ShowDependenciesTask, which="show_upstream"
+    )
     upstream_depencies_sub.add_argument("model_name")
 
     downstream_depencies_sub = subs.add_parser(
         "show_downstream",
         parents=[base_subparser],
         help="Show downstream dependencies for a model",
     )
-    downstream_depencies_sub.set_defaults(cls=show_dependencies_task.ShowDependenciesTask, which="show_downstream")
+    downstream_depencies_sub.set_defaults(
+        cls=show_dependencies_task.ShowDependenciesTask, which="show_downstream"
+    )
     downstream_depencies_sub.add_argument("model_name")
 
+    open_sub = subs.add_parser(
+        "open",
+        parents=[base_subparser],
+        help="Open the a source/compiled/run file for a model",
+    )
+
+    open_sub.set_defaults(cls=open_task.OpenTask, which="open", code_type="compiled")
+
+    open_sub.add_argument("model_name", help="The name of the model to open")
+
+    code_type = open_sub.add_mutually_exclusive_group()
+
+    code_type.add_argument(
+        "--source", "-s", action="store_const", const="source", dest="code_type"
```

Comment:
An earlier version of this had this flag called `--raw`, but I went with `--source` because that's what un-compiled model files are called in `dbt docs`. It also started with a different letter to `--run`, meaning one-letter flags were possible.
I'm a bit wary of introducing a flag called `source` given that `sources`s are now a concept within dbt, any thoughts?

Reply:
🤔  interesting. I don't actually think I know what the difference between `compiled` and `run` is (so at the very least we should lay that out in the docs here somewhere).

I think I like `source` best of these options, since it's more like "source code".

Reply:
Cool. I've updated the help messages and README to be clearer on the difference between `compiled` and `run`.